### ISSUE TITLE
Implement Relationship Importing Functionality for the Shelf POC

### ISSUE THREAD

**Issue by dwhitena** [coralproject/xenia#149]
Based on a config (at first some manual config, then possibly a global Coral Config) import relationships into the graphDB based on the `Item` collection in Mongo.

Also, import/establish arbitrary relationships between items in the `Item` collection.

**Comment by dwhitena** [comment]
See https://github.com/coralproject/shelf-poc

**Comment by dwhitena** [comment]
Specifically, we have tested importing of relationships via the `item-cayley` branch of xenia, which:

1. formats and validates items as they are imported into a mongo collection of items.
2. extracts pre-defined relationships from imported items
3. nests a representation of the pre-defined relationships into imported mongo items.
4. adds quads to the cayley graph DB representing the relationships

Note the following:

- importing is kind of slow.  I tried removing any Cayley related stuff and the importing is still slow, so I think we need to do quite a bit of optimization around importing, validating, etc.
- Cayley imports are much faster in bulk.  We may want to provide a very quick import/validation of items into Mongo, with eventual importing of relationships into Cayley.  That is, we could immediately import items into Mongo and queue up quads to add to Cayley.  Then import the quads periodically and/or when the number of quads reaches a certain number.
